src/Data/Swagger/Internal/Schema/Validation.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE LambdaCase                 #-}
 {-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE NoImplicitPrelude          #-}
```

Comment:
We don't use `NoImplicitPrelude`. I have no strong opinion about it specifically, but I value consistency, and elsewhere we use `Prelude.Compat` and `Prelude` compat imports as own block separated from other imports. For example

```haskell
module Data.Swagger.Internal where

import Prelude ()
import Prelude.Compat

import           Control.Lens             ((&), (.~), (?~))
import           Control.Applicative
import           Data.Aeson
...
```

Reply:
Thanks for bearing with me...  :)

I hope I got it right this time: 5e7113381296796e83845faae79288183ceb3272

Even realized I should turn off CPP again if I don't use it...